config.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  port:
+    description: |
+      Prometheus-edge-hub port
+    type: int
+    default: 9091
+  grpc-port:
+    description: |
+      Prometheus-edge-hub grpc-port
+    type: int
```

Comment:
This default makes little sense to me… If it is 0 the grpc is disabled? Then better a negative default. And the docu needs to be meaningful :-)

Reply:
Removed